cluster-autoscaler/cloudprovider/oci/common/oci_ref.go

```diff
@@ -36,6 +37,7 @@ func NodeToOciRef(n *apiv1.Node) (OciRef, error) {
 		PrivateIPAddress:   getNodeInternalAddress(n),
 		PublicIPAddress:    getNodeExternalAddress(n),
 		Shape:              getNodeShape(n),
+		IsNodeSelfManaged:  n.Labels["oci.oraclecloud.com/node.info.byon"] != "",
```

Comment:
This logic may not work if the customer has manually added this annotation to a managed node and set it to 'false'.

Reply:
I changed this to check if the label value equals true.